### PR TITLE
#BE-4921 Add sub organization name option docs

### PR DESCRIPTION
**Pull request by hiilhan**

### CODE REVIEW

**Comment by burakberkkeskin** [comment]
LGTM!

**Review by burakberkkeskin — APPROVED** [review]
LGTM!

File: docs/marketplace/fastlane/testing-distribution/index.md [hunk 1]
```diff
@@ -47,27 +57,49 @@ fastlane add_plugin appcircle_testing_distribution
 
 ```yml
```
Comment (csonuryilmaz) [inline docs/marketplace/fastlane/testing-distribution/index.md:58]: It should be `ruby`

File: docs/marketplace/fastlane/testing-distribution/index.md [hunk 2]
```diff
@@ -47,27 +57,49 @@ fastlane add_plugin appcircle_testing_distribution
...
   appcircle_testing_distribution(
-    personalAPIToken: "$(AC_PERSONAL_API_TOKEN)",
-    profileName: "$(AC_PROFILE_NAME)",
-    createProfileIfNotExists: Boolean,
-    appPath: "$(AC_APP_PATH)",
-    message: "$(AC_MESSAGE)",
+    personalAPIToken: ENV["AC_ACCESS_TOKEN"],
```
Comment (csonuryilmaz) [inline docs/marketplace/fastlane/testing-distribution/index.md:60]: Change to `AC_PERSONAL_API_TOKEN`

**Pull request merged**